src/components/theme-selector/theme-selector.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { shallowMount, mount } from '@vue/test-utils'
+import { createTestingPinia } from '@pinia/testing'
+
+import ThemeGrid from './theme-grid.vue'
+import ThemeSelector from './theme-selector.vue'
+import ThemeSelectorButton from './theme-selector-button.vue'
+import { defineComponent, shallowRef, ShallowRef } from 'vue'
+
+describe('ThemeSelector', () => {
```

Comment:
Thanks for the added tests here. Is there a special reason you chose UTs for this component?

Reply:
No reason, except it has the same subject: changing the theme.
Just adding some UTs on this component to keep an example if we want to push component testing further in the future. But we should focus more on e2e testing... we'll discuss about that next monday :)